docs/ReleaseNotes.txt

```diff
@@ -13,6 +13,14 @@ Bugs fixed in 3.39.2
 [] Correctly import empty groups from a XML file when an imported group has
 been specified.
 [] Empty groups are now added when merging one PasswordSafe database into another.
+[] List view column widths are no longer reset when dragging columns to different
+positions or columns are added or removed.
+[] The user specified Notes font is now also applied to the Notes column in the
+List view.
+[] Entry Protected sttaus and AutoType string is now correclty displayed in the
```

Comment:
Should be
"Entry Protected state and AutoType string are now correctly displayed in the"